packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  Command,
+  CommandOutput,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import {
+  CancelResponse,
+  ContinueResponse,
+  ParametersGatherer
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Observable } from 'rxjs/Observable';
+import { mkdir } from 'shelljs';
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+import { notificationService } from '../notifications';
+import { CancellableStatusBar, taskViewService } from '../statuses';
+import {
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from './commands';
+
+class ForceApexLogGetExecutor extends SfdxCommandletExecutor<
+  ApexDebugLogIdStartTime
+> {
+  public build(data: ApexDebugLogIdStartTime): Command {
+    return new SfdxCommandBuilder()
+      .withDescription(nls.localize('force_apex_log_get_text'))
+      .withArg('force:apex:log:get')
+      .withFlag('--logid', data.id)
+      .withArg('--json')
```

Comment:
My other comment got hidden because the file was renamed, so I'm commenting again. This is going to output the content of a debug log to the Output console. Have a chat with Nick about that behavior. You can probably add something to CommandExecution so ChannelService can decide whether or not to stream stdout.